syft/execution/role.py

```diff
@@ -10,7 +10,7 @@
 import torch
```

Comment:
Even though `Role` no longer directly references `torch`, this import can't be removed because `Actions` sometimes call methods like `torch.manual_seed()`, which require `torch` to be available in the scope where `eval` runs. Crypten runs into the same issue so we need a better solution for this, but I'll leave that for a future PR.

Reply:
Along related lines, sure would love to get rid of that `eval`.